src/components/HeaderPages/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './styles.css';
+
+export default function TitlePage(props) {
```

Comment:
The file name is HeaderPages and the component name is TitlePage. For consistency this names should be the same. Also I think we should name it as PageHeader.